parts/k8s/addons/kubernetesmasteraddons-elb-svc.yaml

```diff
@@ -1,34 +0,0 @@
----
```

Comment:
This manifest was not being referenced during any of the addons data marshalling/conversion into ARM customData.

Reply:
for reference, we stopped using it intentionally in https://github.com/Azure/aks-engine/pull/1622